plugins/modules/rds_instance.py

```diff
@@ -99,7 +99,8 @@
         type: str
     backup_retention_period:
         description:
-          - The number of days for which automated backups are retained (must be greater or equal to 1).
+          - The number of days for which automated backups are retained (>=1).
+            When set to 0, automated backups will be disabled.
```

Comment:
```suggestion
          - When set to C(0), automated backups will be disabled.
```
Part of me thinks we should also mention the caveat that `Can't be set to 0 if the DB instance is a source to read replicas`  ( https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/rds.html#RDS.Client.create_db_instance )

Reply:
We definitely should mention the read replicas case! Why the `C(0)` btw?

Reply:
Using C() tweaks the formatting a little and formats it as a 'value'.  See also:
https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-within-module-documentation